src/main/java/net/sf/jabref/exporter/AutoSaveManager.java

```diff
@@ -106,6 +107,9 @@ private static boolean autoSave(BasePanel panel) {
         } catch (Throwable ex) {
             ex.printStackTrace();
             return false;
+        } finally {
```

Comment:
Personally, I would separate the pathToBeSavedTo and the loaded database in the saveDatabase method. Then this hack wouldn't be necessary.